Bio/Phylo/PAML/baseml.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, alignment=None, tree=None, working_dir=None, out_file=None):
         Paml.__init__(self, alignment, working_dir, out_file)
         if tree is not None:
             if not os.path.exists(tree):
-                raise IOError("The specified tree file does not exist.")
+                raise OSError("The specified tree file does not exist.")
```

Comment:
It seems that `FileNotFoundError` is a better exception here as it's an OSError subclass that precisely describes the error, e.g.:

```
Python 3.8.1 (default, Dec 21 2019, 20:57:38) 
[GCC 9.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> f = open("nonexistent", "r")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'nonexistent'
```